common/reviews/api/node-core-library.api.md

```diff
@@ -602,6 +606,12 @@ export interface IWaitForExitWithStringOptions extends IWaitForExitOptions {
     encoding: BufferEncoding;
 }
 
+// @public (undocumented)
+export interface IWeightedIterable {
+    // (undocumented)
+    weight?: number;
```

Comment:
You probably want this to be a required property so the overload with `weighted: true` mandates it. I suppose unless you're providing a default weight for elements that don't define it.

I'll read through this new iteration in more detail tomorrow morning.